src/js/intl/strings.js

```diff
@@ -293,7 +292,7 @@ exports.strings = {
     'es_AR': 'Otro commit más, y van...',
     'es_MX': 'Otro commit más, y van...',
     'es_ES': 'Hagamos un commit. Atlanta, Georgia! Escucha WHATUPRG también.',
-    'pt_BR': 'Commitando.. Vai Timão!',
+    'pt_BR': 'Commitando.. Vai Fluzão !',
```

Comment:
I don't understand the motivation for this change. I'm Portuguese, not Brazilian,, but still, I would be able to guess the meaning of Timão, whereas for Fluzão I had to google it (it refers to the Fluminense team in particular, apparently?). I assume there are Brazilians who are not into football who may also not be familiar with "fluzão". And in any case, I'm not sure referring to a specific team (as opposed to a generic term for any team) would be desirable here.

All that said, I am definitely out of the loop here, so feel free to enlighten me :)

Reply:
Yes it is a tiny little team that never won the Libertadores and I thought it would be nice to give them this tiny recognition. :)

Reply:
I suppose this is something for @pcottle to decide — would you be OK to replace a generic term for "team" with a term for a specific team, in this case [Fluminense](https://en.wikipedia.org/wiki/Fluminense_FC)? IMHO it's best to stay neutral in this tool, both to avoid rubbing people the wrong way (who may have strong preferences in terms of teams) and to ensure a more broadly known term is used instead of one that requires specific football knowledge.

Reply:
I was just following the fun of "Go Bears!" and some of the other translations. In Brazil "Timão" has a dual meaning, it's the direct translation of "big team" but it is also known of the team Corinthians with a large number of fans.

Reply:
> In Brazil "Timão" has a dual meaning, it's the direct translation of "big team" but it is also known of the team Corinthians with a large number of fans.

I had this suspicion indeed. In this case I suppose it's a coin toss to use one or the other, even if "Fluzão" is unambiguously referring to a team and the other could be interpreted as "big team" as you said. I won't object to either choice, but I'd like @pcottle to chime in (@maxoliverbr could you unresolve this thread so it's not hidden from view until a decision is made?).